descwl_coadd/interp.py

```diff
@@ -132,6 +133,78 @@ def interp_image_nocheck(image, bad_msk):
     return interp_image
 
 
+class CTInterpolator(object):
+    """
+    A class wrapping the interp_image_nocheck function to do
+    an inplace interp and sets the INTRP bit
+
+    This conforms to the interface required for the interpolator
+    sent to descwl_coadd.coadd.warp_exposures
+
+    This is a "functor" meaning the object can be called
+        interpolator(exposure)
+    """
+    def __init__(self, bad_mask_planes=FLAGS2INTERP):
+        self.bad_mask_planes = bad_mask_planes
```

Comment:
While we are at it, could you make `buff` passed to `_get_nearby_good_pixels` an optional argument to `__init__`?